### ISSUE TITLE
Voting on own solutions or comments

### ISSUE THREAD

**Issue by Seprum** [Codewars/codewars.com#207]
There is an ability to upvote/downvote own published solutions/comments. Is it logical?

**Comment by jhoffner** [comment]
Yes, there was a serious of community discussion about this a few years ago. This behavior mimicks Reddit's system, where by default you upvote your own comment. Since then we changed solutions to have multiple vote options, so we can no longer just automatically add the vote - so in this case it might make sense to prohibit user's voting on their own solutions since it can no longer be the default behavior.

**Comment by pochmann** [comment]
I just noticed this as well. I can upvote my own solutions. Wat. This should really get fixed. Or am I *supposed* to upvote myself, choosing between "Best Practices" and "Clever" (apparently I can't do both)?

**Comment by dinglemouse2250** [comment]
For solutions, it seems most people only vote for what is already voted for. This means the earlier solutions tend to get to the top of those lists even if they don't really deserve to be. So if you *honestly* believe your solution is better than the ones currently marked as best practice / clever then there is really no other way to "advertise" your solution than to vote for it yourself so at least it makes it into the lists of what other people are looking at...
